src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -16203,6 +16205,34 @@ .maxstack  2
   IL_0026:  call       ""int System.Span<int>.Length.get""
   IL_002b:  call       ""void System.Console.Write(int)""
   IL_0030:  ret
+}");
+            comp = CreateCompilationWithMscorlibAndSpan(source, TestOptions.DebugExe);
```

Comment:
>TestOptions.DebugExe [](start = 64, length = 20)

Consider adding some Debug verification or flipping some options in other existing tests.